src/core/Akka.Remote.TestKit/MultiNodeSpec.cs

```diff
@@ -379,6 +379,7 @@ public static Config BaseConfig
                             }
                           }
                         }
+                        cluster.downing-provider-class = """" #disable SBR by default
```

Comment:
Backwards compat for the MNTR - keep the SBR disabled by default. Most tests probably don't want the SBR running anyway unless they explicitly enable it.